### PR TITLE
[linux] install UseMultiArch.cmake file

### PR DESCRIPTION
**Pull request by wsnipex**
needed for binary addons

@stefansaraev @MilhouseVH this should fix building binary addons standalone.

### CODE REVIEW

File: Makefile.in [hunk 1]
```diff
@@ -623,6 +623,9 @@ endif
 	@for f in project/cmake/scripts/common/*.cmake; do \
 	  install -m 0644 $$f $(DESTDIR)$(libdir)/@APP_NAME_LC@; \
 	done
+ifeq ($(findstring linux,@ARCH@),linux)
```
Comment (stefansaraev) [inline Makefile.in]: ifeq ($(findstring Linux,$(shell uname -s)),Linux)

Comment (wsnipex) [inline Makefile.in]: why? ARCH already holds this

Comment (stefansaraev) [inline Makefile.in]: here ARCH holds 'arm'. however, I'll partialy revert #6897 instead..

Comment (wsnipex) [inline Makefile.in]: would host work better for you?
S["host"]="x86_64-unknown-linux-gnu"

Comment (stefansaraev) [inline Makefile.in]: yup. should be fine

however, is there any good reason for not *always* installing UseMultiArch.cmake ?

Comment (wsnipex) [inline Makefile.in]: same reason as why @Montellese wanted it limited to linux in the first place: its only used on linux and we don't want to clutter other platforms.

**Comment by wsnipex** [comment]
updated. jenkins build this please

**Pull request merged**